### PR TITLE
fix: update data-feed-api.test.ts

### PR DESCRIPTION
**Pull request by JulissaDantes**
This branch passed durable test: https://github.com/3box/ceramic-tests/actions/runs/8973949493

Changes include:
- Update the `genesisCommit`
- Un-skip the tests
- Increase timeout

### CODE REVIEW

File: suite/src/__tests__/fast/data-feed-api.test.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ describe.skip('Datafeed SSE Api Test', () => {
       const doc = await genesisCommit(ceramicNode1, modelInstanceDocumentMetadata, false)
       expectedEvents.add(doc.tip.toString())
 
-      await accumulator.waitForEvents(expectedEvents, 1000 * 60)
+      await accumulator.waitForEvents(expectedEvents, 2000 * 60)
```
Comment (stbrody) [inline suite/src/__tests__/fast/data-feed-api.test.ts:80]:
```suggestion
      await accumulator.waitForEvents(expectedEvents, 1000 * 60 * 2)
```

Comment (stbrody) [reply suite/src/__tests__/fast/data-feed-api.test.ts:80]: total unimportant nitpick, but personally I find it easier to mentally parse where these numbers are coming from when written out like this (millis->seconds->minutes)

File: suite/src/__tests__/fast/data-feed-api.test.ts [hunk 2]
```diff
@@ -167,7 +167,7 @@ describe.skip('Datafeed SSE Api Test', () => {
       })
       expectedEvents.add(doc.tip.toString())
       // By waiting for the expected events we confirm the api delivers all events)
-      await accumulator.waitForEvents(expectedEvents, 1000 * 60)
+      await accumulator.waitForEvents(expectedEvents, 2000 * 60)
```
Comment (stbrody) [inline suite/src/__tests__/fast/data-feed-api.test.ts:170]:
```suggestion
      await accumulator.waitForEvents(expectedEvents, 1000 * 60 * 2)
```

File: suite/src/__tests__/fast/data-feed-api.test.ts [hunk 3]
```diff
@@ -15,16 +15,16 @@ import { decode } from 'codeco'
 const ComposeDbUrls = String(process.env.COMPOSEDB_URLS)?.split(',')
 const adminSeeds = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS).split(',')
 
-async function genesisCommit(node: CeramicClient, modelInstanceDocumentMetadata: ModelInstanceDocumentMetadataArgs, anchor: boolean) {
+async function genesisCommit(CeramicNode: CeramicClient, modelInstanceDocumentMetadata: ModelInstanceDocumentMetadataArgs, anchor: boolean) {
```
Comment (stbrody) [inline suite/src/__tests__/fast/data-feed-api.test.ts:18]: another total nitpick, but convention is for variable names to start with lower case, only types generally start with capital letter.

Comment (JulissaDantes) [reply suite/src/__tests__/fast/data-feed-api.test.ts:18]: oh yeah you are right, thanks for noticing

**Review by stbrody — APPROVED** [review]
changes here all seem safe an innocuous, but I'm not convinced that they are actually addressing the root cause of the failures.

**Pull request merged**